### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by evgenkisel** [voxpupuli/puppet-kafka#338]
Hello!

I'd like to have changes from the PR [Support kafka 3.0 topics](https://github.com/voxpupuli/puppet-kafka/pull/337) in my environment.

Please release a new version of the module.

Thank you!

**Comment by zicklam** [comment]
I would like to push this issue to get a new release :)

**Comment by mdklapwijk** [comment]
Yes, please tag and release...

**Comment by Fabian1976** [comment]
Yes, please release a new version which also support puppet 8 (this module works with puppet8. Dependencies and upper limits just need to be updated).
